pkg/kubectl/cmd/util/helpers.go

```diff
@@ -223,6 +224,13 @@ func StandardErrorMessage(err error) (string, bool) {
 			return fmt.Sprintf("The connection to the server %s was refused - did you specify the right host or port?", host), true
 		}
 		return fmt.Sprintf("Unable to connect to the server: %v", t.Err), true
+	case *json.SyntaxError:
+		klog.V(4).Infof("JSON syntax error: %s", t.Error())
+		switch {
+		case strings.Contains(t.Error(), "invalid character"):
```

Comment:
string matching in error messages is fragile... the "connection refused" string comparison above is not a pattern we want to expand